vendor/rbac-lookup/Makefile

```diff
@@ -0,0 +1,27 @@
+
+ifeq ($(OS), darwin)
+  OS=Darwin
+else
+  OS=Linux
+endif
+
+ifeq ($(ARCH), amd64)
+  ARCH=x86_64
```

Comment:
So if arch is x86_64 this would mean we would use i386 binary

Reply:
I was following the pattern from [hugo](https://github.com/cloudposse/packages/blob/master/vendor/hugo/Makefile#L4-L8), which also fails on Windows.